Rakefile

```diff
@@ -5,10 +5,12 @@ $:.unshift File.join(File.dirname(__FILE__), 'lib')
 require 'rubygems' unless ENV['NO_RUBYGEMS']
 
 require 'bundler'
+require 'pry-nav'
```

Comment:
Well... maybe not. It was just helpful when I was debugging tests. Does it do any harm?